Main.java

```diff
@@ -9,7 +10,31 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create a scanner for the user
+    Scanner input = new Scanner(System.in);
+
+    // ask user to enter in the x coordinate of the first point
+    System.out.println("Please enter in the x coordinate of the first point");
+
+    int point1 = input.nextInt();
+
+    System.out.println("Please enter in the y coordinate of the first point");
+
+    int point2 = input.nextInt();
+
+    System.out.println("Please enter in the x coordinate of the second point");
+
+    int point3 = input.nextInt();
+
+    System.out.println("Please enter in the y coordinate of the second point");
+
+    int point4 = input.nextInt();
+    
+    int total1 = point4 - point2;
+    int total2 = point3 - point1;
+
+    double total3 = total1 / total2;
```

Comment:
Make sure your variable names are named after what they are storing. total3 is really the slope. total1 and total2 would be the rise and the run respectively. Also, would be it possible to have a point where the x or y value had a decimal place? They should probably be doubles instead. You have a hint of this because the sample output says 2.0. Also, don't forget to comment your code. You kind of gave up commenting after line 16.